src/components/Votes/VoteDisplay/VoteDisplay.tsx

```diff
@@ -26,37 +26,34 @@ export const VoteDisplay = ({usedVotes, possibleVotes}: VoteDisplayProps) => {
       <span title={t("VoteDisplay.tooltip", {remaining: possibleVotes - usedVotes, total: possibleVotes})}>
         {usedVotes} / {possibleVotes}
       </span>
-      <div className="vote-display__short-actions">
+      <ul className="vote-display__short-actions">
         {isAdmin && (
-          <>
-            <div className="short-actions__short-action">
-              <button aria-label={t("VoteDisplay.abortActionTooltip")} className="short-action__button" onClick={() => store.dispatch(Actions.abortVoting(voting!))}>
-                <CancelIcon />
-              </button>
-              <span className="short-action__tooltip">{t("VoteDisplay.abortActionTooltip")}</span>
-            </div>
-            <div className="short-actions__short-action">
-              <button aria-label={t("VoteDisplay.finishActionTooltip")} className="short-action__button" onClick={() => store.dispatch(Actions.closeVoting(voting!))}>
-                <FlagIcon />
-              </button>
-              <span className="short-action__tooltip">{t("VoteDisplay.finishActionTooltip")}</span>
-            </div>
-          </>
-        )}
-        {!isAdmin && (
-          <div className="short-actions__short-action">
+          <li className="short-actions__short-action">
             <button
-              aria-label={isReady ? t("MenuBars.unmarkAsDone") : t("MenuBars.markAsDone")}
-              className={classNames("short-action__button", {"short-action__button--ready": isReady}, {"short-action__button--unready": !isReady})}
-              onClick={() => store.dispatch(Actions.setUserReadyStatus(me!.user.id, !isReady))}
+              aria-label={t("VoteDisplay.finishActionTooltip")}
+              data-tooltip-id="info-bar__tooltip"
+              data-tooltip-content={t("VoteDisplay.finishActionTooltip")}
+              className="short-action__button"
+              id="vote-display__finish-short-action"
+              onClick={() => store.dispatch(Actions.closeVoting(voting!))}
             >
-              <CheckIcon className="short-action__check-icon" />
-              <CancelIcon className="short-action__cancel-icon" />
+              <FlagIcon />
             </button>
-            <span className="short-action__tooltip">{isReady ? t("MenuBars.unmarkAsDone") : t("MenuBars.markAsDone")}</span>
-          </div>
+          </li>
         )}
-      </div>
+        <li className="short-action__short-actions">
+          <button
+            aria-label={isReady ? t("MenuBars.unmarkAsDone") : t("MenuBars.markAsDone")}
+            data-tooltip-id="info-bar__tooltip"
+            data-tooltip-content={isReady ? t("MenuBars.unmarkAsDone") : t("MenuBars.markAsDone")}
+            className={classNames("short-action__button", {"short-action__button--ready": isReady}, {"short-action__button--unready": !isReady})}
```

Comment:
the differencing in ready and unready seems a bit overcomplicated

Reply:
What do you mean?

Reply:
Shouldn't "unready" be the default state?
So why is there a special modifier for this state in the classname and CSS?